mina-transactions-generator/values.yaml

```diff
@@ -2,44 +2,67 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# -- The number of replicas
 replicaCount: 1
 
 image:
+  # -- The repository of the image
   repository: 673156464838.dkr.ecr.us-west-2.amazonaws.com/mina-transactions-generator
+  # -- The pullPolicy used when pulling the image
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: 0.1.2-5b82cae
+  # -- The tag of the iamge. Overrides the image tag whose default is the chart appVersion.
```

Comment:
suggestion: we could replace `iamge` with `image` 😆

Reply:
Thnaks fro sopting tihs out! Rsevoled!